lib/fog/kubevirt/compute/models/vms.rb

```diff
@@ -24,6 +26,99 @@ def all(filters = {})
         def get(name)
           new service.get_vm(name)
         end
+
+        def create(args = {})
+          vm_name = args.fetch(:vm_name)
+          cpus = args.fetch(:cpus, nil)
+          memory_size = args.fetch(:memory_size)
+          image = args.fetch(:image, nil)
+          pvc = args.fetch(:pvc, nil)
+
+          if image.nil? && pvc.nil?
+            raise ::Fog::Kubevirt::Errors::ValidationError
+          end
+          
+          volume = {}
+
+          if !image.nil?
+            volume = {
+              :name => vm_name,
+              :registryDisk => {
+                :image => image
+              }
+            }
+          else
+            volume = {
+              :name => vm_name,
+              :persistentVolumeClaim => {
+                :claimName => pvc
+              }
+            }
+          end
+
+          vm = {
+            :apiVersion => "kubevirt.io/v1alpha2",
```

Comment:
I think we'd better extract ```kubevirt.io/v1alpha2``` to a constant of shared location
we already have in lib/fog/compute/kubevirt.rb:        
```KUBEVIRT_VERSION = 'v1alpha2'.freeze```